CONTRIBUTING.md

```diff
@@ -3,8 +3,14 @@
 ## Starting dev server
 
 1. Clone this repo or create your own fork and clone it if you want to send PRs
-2. Run `yarn` after cloning the repo to install dependencies
-3. Run `yarn start` which starts the `webpack-dev-server` and `webpack-bundle-analyzer`
+2. Do the following steps:
+cd dejavu/app/src/batteries
```

Comment:
We don't require to navigate till `batteries` path just running git commands in root folder will also work, because path is already mentioned in [.gitmodules](https://github.com/appbaseio/dejavu/blob/dev/.gitmodules#L2)